components/automate-ui/src/app/components/table/table-cell/table-cell.component.scss

```diff
@@ -9,11 +9,17 @@
   border-bottom: 1px solid $chef-grey;
   border-top: 1px solid $chef-grey;
   overflow: hidden;
-  text-overflow: ellipsis;
+  // text-overflow: ellipsis; // TODO: apply this locally
```

Comment:
@susanev - Apoligies, realized I need to loop through this piece with you again

Reply:
i think we should stick with wrapping for now